app/code/Magento/Reports/Controller/Adminhtml/Report/Shopcart.php

```diff
@@ -9,14 +9,23 @@
  *
  * @author      Magento Core Team <core@magentocommerce.com>
  */
+declare(strict_types=1);
+
 namespace Magento\Reports\Controller\Adminhtml\Report;
 
 /**
+ * Class \Magento\Reports\Controller\Adminhtml\Report\Shopcart
+ *
+ * phpcs:disable Magento2.Classes.AbstractApi
  * @api
  * @since 100.0.2
  */
 abstract class Shopcart extends \Magento\Backend\App\Action
 {
+    /**
+     * Authorization of a shop cart report
+     */
+    const ADMIN_RESOURCE = 'Magento_Reports::shopcart';
```

Comment:
It's ok to introduce a constant to this class (as the constant actually already exists as it is inherited from the parent class, unfortunately, SVC tests are not able to identify this currently)

Reply:
@sivaschenko, thank you for advice :)

Reply:
Thank you.